docs/reference/islandora_defaults_reference.md

```diff
@@ -0,0 +1,395 @@
+# Drupal Configuration Provided By Islandora Defaults
+
+This is a list of the configuration items provided by the Islandora Defaults module. 
+
+## Entities
+
+(see Glossary: [Entity](../user-documentation/glossary.md#entity))
+
+### Content Types (i.e. Node bundles)
+
+|Name  					|Description  		 		|Comments							|
+|---					|---					|---								|
+|Repository Item (islandora_object)   	|An item in your Islandora repository.  |Defaults provides only one node type to use with Islandora, and different behaviours are triggered by different field values. If you would like to use another content type with Islandora, it will need Islandora behaviours configured appropriately (see below). 	|
+
+
+### Media Types (i.e. Media bundles)
+
+|Name  					|Description  		 		|Comments							|
+|---					|---					|---								|
+|Audio (audio)   	| A locally hosted audio file.  	| This is Drupal's built-in Audio media type. Islandora Defaults overwrites the field storage so that it is not actually a locally hosted file. The description is misleading.	|
+|Document (document) 	| An uploaded file or document, such as a PDF.		| This is Drupal's built-in Document media type. Islandora Defaults overwrites the field storage so that it is not actually a locally hosted file. The description is misleading.|
+|File (file)		| Use local files for reusable media.			||
+|Image (image)		| Use local images for reusable media.			| This is Drupal's built-in Image media type. Islandora Defaults overwrites the field storage so that it is not actually a locally hosted file. The description is misleading.	|
+|Video (video)		| A locally hosted video file.				| This is Drupal's built-in Video media type. Islandora Defaults overwrites the field storage so that it is not actually a locally hosted file. The description is misleading.	|
+
+### Vocabularies (i.e. Taxonomy bundles)
+
+|Name			|Description		|Comments			|
+|---			|---			|---				|
+|Islandora Access (islandora_access)	|Terms used to limit, restrict or coordinate access	| Not sure if they actually do affect access.	|
+|Islandora Display (islandora_display)	|Terms used to alter how a repository item is viewed. ie. basic image vs large image	| Check if/how this gets populated.|
+
+### Form Modes
+
+The following "Default" forms are configured. It is unclear how/if these include Islandora-specific choices such as ordering/grouping or form widget configuration.
+
+|Type			|Bundle			|Name	|Comment|
+|---			|---			|---	|---	|
+|Node			|Islandora Object	|Default||
+|Media			|Audio			|Default||
+|Media			|Document		|Default||
+|Media			|File			|Default||
+|Media			|Image			|Default||
+|Media			|Video			|Default||
+|Taxonomy Term		|Islandora Display	|Default||
+|Taxonomy Term		|Islandora Media Use	|Default|This vocabulary is not defined in this module.|
```

Comment:
I'd drop the vocabs that aren't defined in defaults or maybe mention that they are defined by the islandora_core_feature